mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java

```diff
@@ -44,14 +44,14 @@ public void fetchInvoices() {
                 new UseCase.UseCaseCallback<FetchInvoices.ResponseValue>() {
                     @Override
                     public void onSuccess(FetchInvoices.ResponseValue response) {
-                        DebugUtil.log("ivoices fetched successfully",
+                        DebugUtil.log("invoices fetched successfully",
                                 response.getInvoiceList().size());
                         mInvoicesView.showInvoices(response.getInvoiceList());
                     }
 
                     @Override
                     public void onError(String message) {
-                        DebugUtil.log("unable to fetvh invoices");
+                        DebugUtil.log("unable to fetch invoices");
```

Comment:
Do we need these log statements?

Reply:
Yup i think its necessary.They were already there i just fixed the typo.

Reply:
Log statements are only necessary for the developer while testing/debugging the app. Please remove them. I think we were merged by mistake.

Reply:
Done